### PR TITLE
refactor: add default token list

### PR DESCRIPTION
**Pull request by octocat512**

### CODE REVIEW

File: src/store/lists/tokenlist.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
+{
+  "name": "Balanced Token",
```
Comment (parrot9design) [inline src/store/lists/tokenlist.json:2]:
```suggestion
  "name": "Balanced Tokens",
```

File: src/store/lists/tokenlist.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      "chainId": 7,
+      "name": "Stably USD",
+      "symbol": "USDS",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/balancednetwork/icons/master/tokens/usds.png"
+    },
+    {
+      "address": "cx0fa7815de5b2be6e51dc52caa0dc556012ae0f98",
+      "chainId": 7,
+      "name": "OMM Token",
```
Comment (parrot9design) [inline src/store/lists/tokenlist.json:152]:
```suggestion
      "name": "Omm Token",
```

File: src/store/lists/tokenlist.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      "chainId": 1,
+      "name": "GangstaBet Token",
+      "symbol": "GBET",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/balancednetwork/icons/master/tokens/gbet.png"
+    },
+    {
+      "address": "cx2aa9b28a657e3121b75d3d4fe65e569398645d56",
+      "chainId": 1,
+      "name": "Test Token",
```
Comment (parrot9design) [inline src/store/lists/tokenlist.json:96]: >"name": "Test Token",

Is this a placeholder? It's not actually called Test Token, is it?

**Review by parrot9design — APPROVED** [review]

File: src/app/components/SearchModal/filtering.ts [hunk 4]
```diff
@@ -35,11 +35,11 @@ export function createTokenFilterFunction<T extends Token | TokenInfo>(search: s
     return lowerSearchParts.every(p => p.length === 0 || sParts.some(sp => sp.indexOf(p) >= 0));
   };
 
-  return ({ name, symbol, searchableTerms }: T): boolean =>
+  // consider whether the searchableTerms is useful
```
Comment (hetfly) [inline src/app/components/SearchModal/filtering.ts:38]: I guess we can leave this out for now and revisit later when there are more tokens and some additional search logic would be desired. Token symbol and name is good enough. @parrot9design any strong preference here?

File: src/constants/tokens.ts [hunk 5]
```diff
@@ -1,454 +1,130 @@
...
-  'CFT',
-  'Craft',
-);
+// disable prettier printWidth rule
+// prettier-ignore
+export const bnUSD: TokenMap = {
+  [ChainId.MAINNET]: new Token(ChainId.MAINNET, 'cx88fd7df7ddff82f7cc735c871dc519838cb235bb', 18, 'bnUSD', 'Balanced Dollar', 'stable dollar'),
+  [ChainId.SEJONG]: new Token(ChainId.SEJONG, 'cx5838cb516d6156a060f90e9a3de92381331ff024', 18, 'bnUSD', 'Balanced Dollar'),
+  [ChainId.YEOUIDO]: new Token(ChainId.YEOUIDO, 'cx7bd90c91db9b0be9f688442dce7569aebb1ff7fe', 18, 'bnUSD', 'Balanced Dollar'),
+  [ChainId.BERLIN]: new Token(ChainId.BERLIN, 'cx1cd2da25f9942fda5144e139bbda3e5108d3c083', 18, 'bnUSD', 'Balanced Dollar'),
```
Comment (hetfly) [inline src/constants/tokens.ts:52]: can we use token addresses from balanced-js npm package?

File: src/queries/swap/index.ts [hunk 6]
```diff
@@ -17,20 +17,27 @@ type BarType = { time: number; open: number; close: number; high: number; low: n
...
-    const [pair, inverse] = getTradePair(
-      currencies[Field.INPUT]?.symbol as string,
-      currencies[Field.OUTPUT]?.symbol as string,
+    const data = await bnJs.Multicall.getPoolStatsForPair(
+      (currencies[Field.INPUT] as Token)?.address,
+      (currencies[Field.OUTPUT] as Token)?.address,
     );
-    if (pair && !isQueue(pair)) {
+
+    console.log('data', data);
```
Comment (hetfly) [inline src/queries/swap/index.ts:25]: remove console log

**Comment by hetfly** [comment]
Also I would prefer other default swap than IUSDC/BALN
image

File: src/app/components/SearchModal/filtering.ts (hunk 4 above)
Comment (parrot9design) [reply src/app/components/SearchModal/filtering.ts:38]: Name and symbol should be enough for now. Does the current token list include any additional search logic that this PR has missed?

**Comment by parrot9design** [comment]
> Also I would prefer other default swap than IUSDC/BALN image

Yeah, let's keep it as BALN -> bnUSD, unless @BennyOptions has another preference? It's not a trade we like to make often, but it is the best chart to highlight.

File: src/app/components/SearchModal/filtering.ts (hunk 4 above)
Comment (octocat512) [reply src/app/components/SearchModal/filtering.ts:38]: The token list doesn't have the logic.

**Review by hetfly — APPROVED** [review]

**Comment by BennyOptions** [comment]
Is this good to merge? @octocat512 please merge if it is

**Pull request merged**